src/vos/evt_priv.h

```diff
@@ -410,14 +412,14 @@ evt_node_read_rect_at(struct evt_context *tcx, struct evt_node *node,
  * Returns the rectangle at the index
  */
 static inline void
-evt_nd_off_read_rect_at(struct evt_context *tcx, umem_off_t nd_off,
+evt_nd_off_rect_read_at(struct evt_context *tcx, umem_off_t nd_off,
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?